### PR TITLE
Added example querying number of timestamp bits on suggestion from Fl…

### PR DESCRIPTION
**Pull request by kenrussell**
…orian Boesch.

### CODE REVIEW

**Comment by kenrussell** [comment]
@zhenyao @grorg @jdashg please review.

File: extensions/EXT_disjoint_timer_query/extension.xml [hunk 1]
```diff
@@ -254,6 +253,69 @@
...
+        var timeElapsedQuery;
+        var startQuery;
+        var endQuery;
+
+        var useTimestamps = false;
+
+        if (ext.getQueryEXT(ext.TIMESTAMP_EXT, ext.QUERY_COUNTER_BITS_EXT) > 0) {
+          useTimestamps = true;
+        }
+
```
Comment (jdashg) [inline extensions/EXT_disjoint_timer_query/extension.xml]: Clear gl.getParameter(ext.GPU_DISJOINT_EXT) right before starting queries.
It's technically cleared in the previous example, but it's good for this example to be self-contained.

Comment (kenrussell) [inline extensions/EXT_disjoint_timer_query/extension.xml]: Thanks. Making this change now.

**Comment by kenrussell** [comment]
Merging now.

**Pull request merged**